apps/wallet-dashboard/sentry.common.config.mjs

```diff
@@ -0,0 +1,9 @@
+export const IS_PROD = process.env.NODE_ENV === 'production';
+
+export const SENTRY_DSN = IS_PROD
+    ? 'https://cb83626ca07d6cf66ca2f901cf53c051@o4508279186718720.ingest.de.sentry.io/4508647247249488'
+    : 'https://ba5d6596291f12e88625e02eb942b742@o4508279186718720.ingest.de.sentry.io/4508647248691280';
+
+export const SENTRY_PROJECT_NAME = IS_PROD ? 'iota-wallet-dashboard' : 'iota-wallet-dashboard-dev';
+export const SENTRY_ORG_NAME = 'iota-foundation-eu';
+export const SENTRY_TRACES_RATE = 1;
```

Comment:
We use `0.0025` in the wallet because we fetch balances so often that it would be a crazy logging if we log 100% of the requests made with the sentry client

Reply:
Actually, lets use the rate from the wallet from apps backend because the refresh rates are basically the same

Reply:
I tried to use the traces rate from growthbook but I seem unable to, nextjs complains about not being able compile some files